daemon/daemon_aufs.go

```diff
@@ -1,4 +1,4 @@
-// +build !exclude_graphdriver_aufs
+// +build !exclude_graphdriver_aufs,!windows
```

Comment:
@jhowardmsft Perhaps we should make this `!exclude_graphdriver_aufs,linux` to make it clear that this must build only when not explicitly excluded AND when the system is Linux. This applies to the others as well.